developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -159,11 +159,13 @@ You'll have to install the **new** Mongo PHP extension_ and enable it:
     $ sudo phpenmod mongodb
 
 .. _adapter: https://github.com/alcaeus/mongo-php-adapter
+.. _composer: https://getcomposer.org
 
-Finally, you have to install the Mongo PHP adapter_:
+Finally, you have to install the Mongo PHP adapter_ using the dependency manager composer_:
 
 .. code-block:: bash
 
+    $ sudo apt-get install composer
```

Comment:
That is not the same problem. Composer is independent and can/should be the most recent possible.
php-mongo is a PHP extension and must be installed along other PHP extensions. The most secure way to do that is to use packages that are made to run together. Now, if you want more recent packages for PHP and its extensions, you can use an external repository, like https://deb.sury.org/ (it is made by the official maintainer of PHP for Debian, so trustable).